bitnami/dremio/README.md

```diff
@@ -0,0 +1,1288 @@
+
+```yaml
+dremio:
+  auth:
+    enabled: false
+  dremioConf:
+    configOverrides:
+      services.coordinator.web.auth.type: "azuread"
+      services.coordinator.web.auth.config: "/opt/bitnami/dremio/conf/azuread.json"
+      extraSecretFiles:
+        azuread.json: |
+          {
+            "oAuthConfig": {
+              "clientId": "<clientId>",
+              "clientSecret": "<clientSecret>",
+              "redirectUrl": "https://<dremio.host>:9047/sso",
+              "authorityUrl": "https://login.microsoftonline.com/<directory.id>/v2.0",
+              "scope": "openid profile",
+              "jwtClaims": {
+                "userName": "preferred_username"
+              }
+            }
+          }
+```
+
+### External Zookeeper support
+
+You may want to have Dremio connect to an external zookeeper rather than installing one inside your cluster. Typical reasons for this are to use a managed database service, or to share a common database server for all your applications. To achieve this, the chart allows you to specify credentials for an external database with the [`externalZookeeper` parameter](#parameters). You should also disable the Zookeeper installation with the `zookeeper.enabled` option. Here is an example:
+
+```console
+zookeper.enabled=false
+externalZookeeper.hosts[0]=myexternalhost
+externalZookeeper.port=2181
+```
+
+### TLS secrets
+
+TLS support for the Web interface can be enabled in the chart by specifying the `dremio.tls.` parameters while creating a release. The following parameters should be configured to properly enable the TLS support in the cluster:
+
+- `dremio.tls.enabled`: Enable TLS support. Defaults to `false`
+- `dremio.tls.usePemCerts`: Use PEM certificates instead of a JKS file.
+- `dremio.tls.existingSecret`: Name of the secret that contains the certificates. No defaults.
+
+When `dremio.tls.usePemCerts=true`:
+
+- `dremio.tls.certFilename`: Certificate filename. No defaults.
+- `dremio.tls.certKeyFilename`: Certificate key filename. No defaults.
+- `dremio.tls.certCAFilename`: CA Certificate filename. No defaults.
+
+When `dremio.tls.usePemCerts=false`:
+
+- `dremio.tls.keystoreFilename`: Certificate filename. No defaults.
+
+For example:
+
+First, create the secret with the certificates files:
+
+```console
+kubectl create secret generic certificates-tls-secret --from-file=./cert.pem --from-file=./cert.key --from-file=./ca.pem
+```
+
+Then, use the following parameters:
+
+```console
+dremio.tls.enabled="true"
+dremio.tls.existingSecret="certificates-tls-secret"
+dremio.tls.usePemCerts="true"
+dremio.tls.certFilename="cert.pem"
+dremio.tls.certKeyFilename="cert.key"
+dremio.tls.certCAFilename="ca.pem"
+```
+
+It is also possible to rely on the chart certificate auto-generation capabilities. The chart supports two different ways to auto-generate the required certificates:
```

Comment:
Maybe it would we more clear stating at the beginning of the section that you have 3 options:
- Autogenerating
- Providing PEM files
- Providing a JKS keystore

and then detail each of the options.